package/firmware/brcmfmac-sdio-firmware/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (C) 2019-2020 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=brcmfmac-sdio-firmware
+PKG_SOURCE_DATE:=2022-07-06
+PKG_SOURCE_VERSION:=afc477e807c407736cfaff6a6188d09197dfbceb
+PKG_MIRROR_HASH:=1afabfe429163bd87a03d8305352a8df9a25d59ae6ba600f1955bbd58e17f9a5
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/LibreELEC/brcmfmac_sdio-firmware
+
+PKG_MAINTAINER:=Florian Fainelli <f.fainelli@gmail.com>
+PKG_LICENSE_FILES:=LICENCE.broadcom_bcm43xx
```

Comment:
Where do I find this license file?

Reply:
There is a missing commit that imports that license into the LICENSE folder, I will amend the pull request.